### ISSUE TITLE
Test errors with numpy 1.11.1rc1

### ISSUE THREAD

**Issue by cgohlke** [matplotlib/matplotlib#6489]
On Windows, Python 3.5.1, 64-bit:
```
numpy 1.11.1rc1
matplotlib 1.5.2rc2
<snip>
======================================================================
ERROR: test suite for <class 'matplotlib.tests.test_axes.transplant_class.<locals>.C'>
----------------------------------------------------------------------
Traceback (most recent call last):
  File "X:\Python35\lib\site-packages\nose\suite.py", line 210, in run
    self.setUp()
  File "X:\Python35\lib\site-packages\nose\suite.py", line 293, in setUp
    self.setupContext(ancestor)
  File "X:\Python35\lib\site-packages\nose\suite.py", line 316, in setupContext
    try_run(context, names)
  File "X:\Python35\lib\site-packages\nose\util.py", line 471, in try_run
    return func()
  File "X:\Python35\lib\site-packages\matplotlib\testing\decorators.py", line 154, in setup_class
    cls._func()
  File "X:\Python35\lib\site-packages\matplotlib\tests\test_axes.py", line 87, in test_formatter_large_small
    if tuple(map(int, np.__version__.split('.'))) >= (1, 11, 0):
ValueError: invalid literal for int() with base 10: '1rc1'

======================================================================
ERROR: test suite for <class 'matplotlib.tests.test_bbox_tight.transplant_class.<locals>.C'>
----------------------------------------------------------------------
Traceback (most recent call last):
  File "X:\Python35\lib\site-packages\nose\suite.py", line 210, in run
    self.setUp()
  File "X:\Python35\lib\site-packages\nose\suite.py", line 293, in setUp
    self.setupContext(ancestor)
  File "X:\Python35\lib\site-packages\nose\suite.py", line 316, in setupContext
    try_run(context, names)
  File "X:\Python35\lib\site-packages\nose\util.py", line 471, in try_run
    return func()
  File "X:\Python35\lib\site-packages\matplotlib\testing\decorators.py", line 154, in setup_class
    cls._func()
  File "X:\Python35\lib\site-packages\matplotlib\tests\test_bbox_tight.py", line 94, in test_bbox_inches_tight_raster
    if tuple(map(int, np.__version__.split('.'))) >= (1, 11, 0):
ValueError: invalid literal for int() with base 10: '1rc1'

----------------------------------------------------------------------
```

**Comment by tacaswell** [comment]
This was added to skip tests that fail due to fixing of a numpy bug.  We fix the tests for real on 2.x on, but did not backport those changes.

attn @jenshnielsen 

We should use loose version here.

**Comment by jenshnielsen** [comment]
Yes we should use LooseVersion. I was lazy and copied your solution from test_axes:test_formatter_large_small in https://github.com/matplotlib/matplotlib/commit/26cba4b546717b4040008ffc292a020496b95371 yesterday. I have fixed both of them now on #6490

**Comment by tacaswell** [comment]
closed by #6490
